inspirehep/modules/workflows/workflows/article.py

```diff
@@ -219,19 +212,28 @@
 
 CHECK_IF_SUBMISSION_AND_ASK_FOR_APPROVAL = [
     IF_ELSE(
-        is_record_relevant,
-        [halt_record(
-            action="hep_approval",
-            message="Submission halted for curator approval.",
-        )],
+        is_record_merged,
         [
-            reject_record("Article automatically rejected"),
-            stop_processing
+            IF(
+                are_there_conflicts,
+                [submit_to_the_curator]  # for the manual resolution of the conflicts
+            )
+        ], [
+            IF_ELSE(
+                is_record_relevant,
+                [halt_record(
+                    action="hep_approval",
+                    message="Submission halted for curator approval.",
+                )],
+                [
+                    reject_record("Article automatically rejected"),
+                    stop_processing
+                ]
+            )
         ]
-    ),
+    )
```

Comment:
@kaplun I changed this part of the workflow in order to:

-  Do not block the workflow if we are merging 2 articles and there are no conflicts
-  Block the workflow if we are merging 2 articles and there are conflicts
-  Maintain the same behaviour for new articles

Let me know if it makes sense for you